test/src/main/java/org/apache/accumulo/test/shell/ShellServerIT.java

```diff
@@ -1584,19 +1583,33 @@ public void verifyPerTableClasspath(final String table, final File fooConstraint
         "config -t " + table + " -s " + Property.TABLE_CLASSLOADER_CONTEXT.getKey() + "=" + context,
         true);
 
-    sleepUninterruptibly(250, TimeUnit.MILLISECONDS);
-
+    try {
+      Thread.sleep(250);
+    } catch (InterruptedException ex) {
```

Comment:
This could probably just be allowed to fall outside the `@Test` method that called this helper method. Same with the other new catch blocks in this file.

Reply:
Modified in 4ae251cf0a